tests/functional_tests/test__test_project.py

```diff
@@ -71,7 +71,7 @@ def setUp(self):
         for _, directories, files in os.walk(self.memStatsFolder):
             # The result folder shall have 0 subdirectories and three summary files
             self.assertEqual(len(directories), 0)
-            self.assertEqual(len(files), 4)
+            self.assertEqual(len(files), 3)
```

Comment:
This dependent of the `--teamscale` argument.
We could write another test for this where it is activated.